logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/debug/DebugScope.kt

```diff
@@ -125,14 +126,18 @@ class DebugScope internal constructor(
 
     private val messageEnvelopeCreator: MessageEnvelopeCreator
         get() = MessageEnvelopeCreatorImpl(
+            conversationRepository = conversationRepository,
+            legalHoldStatusMapper = LegalHoldStatusMapperImpl(),
```

Comment:
Question: There is maybe no big gain, but can we have `LegalHoldStatusMapperImpl` as a singleton ? Either though `MapperProvider` or as an object instead of a class ? :D